Lib/ufo2ft/featureWriters/__main__.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+import logging
+from io import StringIO
+
+from fontTools.misc.cliTools import makeOutputFileName
+
+from ufo2ft.constants import FEATURE_WRITERS_KEY
+from ufo2ft.featureCompiler import FeatureCompiler
+from ufo2ft.featureWriters import loadFeatureWriterFromString, logger
+
+try:
+    import ufoLib2
+
+    loader = ufoLib2.Font
+except ImportError:
+    import defcon
+
+    loader = defcon.Font
+
+logging.basicConfig(level=logging.INFO)
+
+parser = argparse.ArgumentParser(description="Apply feature writers to a UFO file")
+parser.add_argument("--output", "-o", metavar="OUTPUT", help="output file name")
+parser.add_argument("ufo", metavar="UFO", help="UFO file")
+parser.add_argument(
+    "writers",
+    metavar="WRITER",
+    nargs="*",
+    help="list of feature writers to enable",
+)
+
+args = parser.parse_args()
+if not args.output:
+    args.output = makeOutputFileName(args.ufo)
+
+ufo = loader(args.ufo)
+writers = [loadFeatureWriterFromString(w) for w in args.writers]
+compiler = FeatureCompiler(ufo, featureWriters=writers or None)
+compiler.setupFeatures()
+buf = StringIO()
+compiler.writeFeatures(buf)
+ufo.features.text = buf.getvalue()
+
+if FEATURE_WRITERS_KEY in ufo.lib:
+    del ufo.lib[FEATURE_WRITERS_KEY]
```

Comment:
There should be a flag to keep these in the UFO lib. One may want to write a feature to UFO but keep some other features for compilation time.

Reply:
let's keep this simple for now instead of adding options. I see it more for debugging than for use in prod. So let's instead keep the lib as is, and if one wants to build off that ufo, one has to remember to delete any existing featureWriters from the lib to so that compile is idempotent